tests/auth_tests/test_decorators.py

```diff
@@ -142,3 +146,37 @@ def a_view(request):
         request.user = self.user
         with self.assertRaises(PermissionDenied):
             a_view(request)
+
+
+class UserPassesTestDecoratorTest(TestCase):
+    factory = RequestFactory()
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.user_pass = models.User.objects.create(username="joe", password="qwerty")
+        cls.user_deny = models.User.objects.create(username="jim", password="qwerty")
+        models.Group.objects.create(name="Joe group")
+        # Add permissions auth.add_customuser and auth.change_customuser
+        perms = models.Permission.objects.filter(
+            codename__in=("add_customuser", "change_customuser")
+        )
+        cls.user_pass.user_permissions.add(*perms)
```

Comment:
What's the goal of creating these instances in `setUpTestData` if there is a single test? Do you expect more tests to be added?